### ISSUE TITLE
Screen readers don't announce total elements in a virtualized list

### ISSUE THREAD

**Issue by hross** [OfficeDev/office-ui-fabric-react#1366]
We did some testing with a screen reader user today using different list implementations:

•	[Option A – Simple list using aria markup](http://codepen.io/pogostick_snowman/full/QpVQra?renderType=list)
•	[Option B – Simple list using semantic HTML](http://codepen.io/pogostick_snowman/full/QpVQra?renderType=semantic-list)
•	[Option C – Table using aria markup](http://codepen.io/pogostick_snowman/full/QpVQra?renderType=detailslist)
•	[Option D – Table using semantic markup](http://codepen.io/pogostick_snowman/full/QpVQra?renderType=semantic-table)

What came out of this is that the virtualized list and detailslists only announce the number of elements currently present in their implementations (which makes sense based on the markup) -- see options A and C.

I'm not sure if I would qualify this as a bug, per se, except to say that we should at least announce to screen reader users that not all of the elements are present in the list and that they need to scroll to the bottom for more.

**Comment by betrue-final-final** [comment]
@Chris<mailto:cschle@microsoft.com>


________________________________

**Comment by micahgodbolt** [comment]
Is it possible to inform the screen reader of the number  of items in the list through some aria label?

**Comment by hross** [comment]
I think it's possible but I don't know what the most practical/accepted way is to do this. Where does the label go? Maybe it should be the first item in the list so users notice it? But that would be pretty weird. Otherwise I don't know that you can put an aria-label directly on a list and it will work.

We (workflows team in VSTS) are going to do some testing here in the next day or so with some of these codepens so I will update this ticket with what we find out.
